desc/plotting.py

```diff
@@ -3836,3 +3837,110 @@ def plot_logo(save_path=None, **kwargs):
         fig.savefig(save_path, facecolor=fig.get_facecolor(), edgecolor="none")
 
     return fig, ax
+
+
+def plot_gammac(eq, rhos=None, alphas=None, num_pitch=None, **kwargs):
```

Comment:
should also have `ax` and `return_data` options like the other plotting functions.